MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -2130,6 +2238,24 @@
 			productReference = C5ADFBD21DDCC7840011824B /* MapboxCoreNavigationTests.xctest */;
 			productType = "com.apple.product-type.bundle.unit-test";
 		};
+		E2D1494D265CF97D008135A3 /* CarPlayTestHelper */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = E2D14953265CF97D008135A3 /* Build configuration list for PBXNativeTarget "CarPlayTestHelper" */;
+			buildPhases = (
+				E2D14949265CF97D008135A3 /* Headers */,
+				E2D1494A265CF97D008135A3 /* Sources */,
+				E2D1494B265CF97D008135A3 /* Frameworks */,
+				E2D1494C265CF97D008135A3 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = CarPlayTestHelper;
```

Comment:
If I understand correctly, this target and framework exists for running MapboxNavigationTests under SPM; do we even need it for running the tests under Carthage? The target seems to exist solely for some Objective-C code. It’s a similar story with CMapboxCoreNavigation and CTestHelper, but we simplified things a bit by keeping the targets and products unified under Carthage with an alternative umbrella header that SPM ignores.

Reply:
I added this target to the Xcode project so that the imports like https://github.com/mapbox/mapbox-navigation-ios/blob/270bf280af4edc3d5bcec0043d4e5b18f2f740a6/Tests/MapboxNavigationTests/CarPlayNavigationViewControllerTests.swift#L8 would work in both Carthage and SPM. I can wrap them in `#if SWIFT_PACKAGE`, but it seems to bring more burden for future changes. You must not forget to add this ifdef when writing code using the SPM package in Xcode.